utilities/post_migration.py

```
@@ -1572,6 +1654,34 @@ def check_vms(
+            plan.get("enable_nested_virtualization") is False
+            and source_provider.type != Provider.ProviderType.OPENSHIFT
+        ):
+            try:
+                check_cpu_features(
+                    destination_vm=destination_vm,
+                    expected_features=[
+                        {"name": "vmx", "policy": "disable"},
+                        {"name": "svm", "policy": "disable"},
+                    ],
+                )
+            except Exception as exp:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)